engine/base.ftl

```diff
@@ -721,7 +721,8 @@ are added.
                     "Values" : [],
                     "Children" : [],
                     "SubObjects" : false,
-                    "PopulateMissingChildren" : true
+                    "PopulateMissingChildren" : true,
+                    "AttributeSet" : ""
```

Comment:
Also needs an entry for Component?

Reply:
updated in https://github.com/hamlet-io/engine/pull/1502/commits/8e88a00f6a4cd65b69bf555e6c2325a03f45d7ed, cheers